configure.ac

```diff
@@ -4559,7 +4571,7 @@ dnl BeOS support removed after SDL 2.0.1. Haiku still works.  --ryan.
 
         # Set up the core platform files
         SOURCES="$SOURCES $srcdir/src/core/os2/*.c"
-        if test x$ac_cv_func_iconv != xyes -o x$ac_cv_header_iconv_h != xyes; then
+        if test x$enable_iconv = xyes -a ( x$ac_cv_func_iconv != xyes -o x$ac_cv_header_iconv_h != xyes ); then
             SOURCES="$SOURCES $srcdir/src/core/os2/geniconv/*.c"
         fi
```

Comment:
@sezero 
Will os2 still build? No missing symbols?

Reply:
It should - no reason otherwise. (Don't have a way of build testing against klibc now, though.)